src/Microsoft.AspNetCore.Razor.Language/Legacy/HtmlTokenizer.cs

```diff
@@ -21,24 +22,24 @@ public HtmlTokenizer(ITextDocument source)
 
         private new HtmlTokenizerState? CurrentState => (HtmlTokenizerState?)base.CurrentState;
 
-        public override HtmlTokenType RazorCommentType
+        public override SyntaxKind RazorCommentKind
         {
-            get { return HtmlTokenType.RazorComment; }
+            get { return SyntaxKind.RazorComment; }
         }
 
-        public override HtmlTokenType RazorCommentTransitionType
+        public override SyntaxKind RazorCommentTransitionKind
         {
-            get { return HtmlTokenType.RazorCommentTransition; }
+            get { return SyntaxKind.RazorCommentTransition; }
         }
 
-        public override HtmlTokenType RazorCommentStarType
+        public override SyntaxKind RazorCommentStarKind
         {
-            get { return HtmlTokenType.RazorCommentStar; }
+            get { return SyntaxKind.RazorCommentStar; }
```

Comment:
No. They are only used by the Tokenizer and the Parser. @NTaylorMullen, correct me if I am wrong but I think these tokens were abstracted because during the era when Razor supported VB.